handler_example_test.go

```diff
@@ -42,6 +44,46 @@ func (*ExamplePingServer) Ping(
 	), nil
 }
 
+// Sum implements pingv1connect.PingServiceHandler.
+func (p *ExamplePingServer) Sum(ctx context.Context, stream *connect.ClientStream[pingv1.SumRequest]) (*connect.Response[pingv1.SumResponse], error) {
+	var sum int64
+	for stream.Receive() {
+		sum += stream.Msg().Number
+	}
+	if stream.Err() != nil {
+		return nil, stream.Err()
+	}
+	response := connect.NewResponse(&pingv1.SumResponse{Sum: sum})
+	return response, nil
```

Comment:
This will be a public example - can we `return connect.NewResponse(....), nil` instead? I do really like that we'll now have an example implementation of all RPC types in the GoDoc.